app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -184,7 +190,7 @@ The following suggestions and resources may be useful:
 * open an account with a digital bank based in the UK – you can do this before you arrive in the UK, and you will not need a UK address to do so
 * [find somewhere to live](https://www.gov.uk/government/publications/how-to-rent)
 * [get a national insurance (NI) number](https://www.gov.uk/national-insurance/your-national-insurance-number)
-* [register with a doctor](https://www.nhs.uk/nhs-services/gps/how-to-register-with-a-gp-surgery/)
+* [register with a doctor through the NHS](https://www.nhs.uk/nhs-services/gps/how-to-register-with-a-gp-surgery/)
 * [find childcare](https://www.gov.uk/get-childcare)
 * [learn about tax](https://www.gov.uk/income-tax)
 * learn about [indefinite leave to remain](https://www.gov.uk/indefinite-leave-to-remain)
```

Comment:
Should the indefinite leave to remain bullet point be one link so it's consistent with the ones above?